README.md

```diff
@@ -1,73 +1,46 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+## To Do List: A To Do list build without frameworks.
 
-> Description the project.
+>This is project built to show the Today's tasks list, and give users access to add new task or delete the completed tasks
 
+## [Deployed Version]()
 
 ## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+* HTML
+* Css
+* Es6
+* Webpack
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
+- To get a local copy open and running follow these simple example steps.
 
-### Install
+   ## Setup
 
-### Usage
+   - Download or Clone Repository.
```

Comment:
- [ ] Kindly add detailed instructions on how to clone the repo and `cd` into the directory.